wdio.conf.mpcc.android.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+const { mpccBaseConfig } = require('./wdio.conf.mpccbase');
+
+exports.config = {
+    ...mpccBaseConfig,
+    capabilities: [
+        {
+            platformName: 'Android',
+            'appium:orientation': 'PORTRAIT',
+            'appium:automationName': 'UiAutomator2',
+            'appium:deviceName': 'emulator-5554',
+            'appium:app': '<path to Android test app>',
```

Comment:
nit: add a TODO if there's an action that needs to be done to replace it with the correct path